app/assets/javascripts/autocise_textarea.js

```diff
@@ -0,0 +1,53 @@
+// Get the max height and apply
+// to all textareas when page loads/reloads
+
+$(document).ready(function() {
+  var numberArray = [];
+  $('textarea').each(function(textarea) {
+    $(this).height( $(this)[0].scrollHeight );
+    numberArray.push($(this).innerHeight());
+    var max = Math.max(...numberArray);
+    $(this).innerHeight(max);
+  });
+});
+
+// Use an event to listen to all input events and then
+// filter out ones that are not on a textarea
+// if the element is a textarea we will call a
+// autoExpandTextarea function and pass in
+// the element as an argument using event.target.
+document.addEventListener('input', function(event) {
+  if (event.target.tagName.toLowerCase() !== 'textarea') return;
+  autoExpandTextarea(event.target);
+}, false);
+
+var autoExpandTextarea = function(field) {
+  // check the field parameter class so we can give the
+  // same height to other textareas with same class.
+  var isMain = field.classList.contains('item-name');
+  var isMaori = field.classList.contains('item_maori_name');
+
+  // Reset field height
+  field.style.height = 'inherit';
```

Comment:
You can use `.css()` in jQuery. Here's what that looks like:
```
field.css({height: 'inherit'});
```

Reply:
This also looks like you're setting a default with this line, which should _probably_ be done in CSS. Then you can use JS to override the rule. Rather than overriding it twice.

Reply:
its javascript